### PR TITLE
add codecov

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #    pip-compile --output-file=requirements.txt requirements.in
 #
 click==7.0                # via flask
+codecov==2.0.15
```
Comment (samuzzal-choudhury) [inline requirements.txt:8]: @miteshvp you will need a codecov token to be created for this repo https://github.com/fabric8-analytics/metrics-accumulator/blob/master/runtests.sh#L70. @tisnik can help you get one if you have not done yet.

**Review by samuzzal-choudhury — APPROVED** [review]
LGTM

File: requirements.txt (hunk 1 above)
Comment (tisnik) [reply requirements.txt:8]: @samuzzal-choudhury we already have the token and the codecov is setup for this repository https://codecov.io/gh/fabric8-analytics/metrics-accumulator

**Review by tisnik — APPROVED** [review]
LGTM, thanks @miteshvp and @samuzzal-choudhury

**Pull request merged**